commons/connector/fitbit/fitbit_intraday_breathing_rate.avsc

```diff
@@ -0,0 +1,14 @@
+{
+  "namespace": "org.radarcns.connector.fitbit",
+  "type": "record",
+  "name": "FitbitIntradayBreathingRate",
```

Comment:
This doesn't look intraday? They named it strangely at Fitbit, because the endpoint only returns a single value per day, not multiple values "intra" a day... Perhaps rename to `FitbitBreathingRate` and remove the `Summary` suffix from the fields.

Reply:
Done.